src/Abp.Web.Resources/Abp/Framework/scripts/libs/abp.signalr-client.js

```diff
@@ -15,13 +15,28 @@ var abp = abp || {};
         // Set the common hub
         abp.signalr.hubs.common = connection;
 
+        let reconnectTime = 5000;
```

Comment:
instead of defining private variables, we can define `abp.signalr.reconnectTime` and `abp.signalr.maxTries` so that developer can configure those values.